### PR TITLE
Tests for Service Update

### PR DESCRIPTION
**Pull request by dashakhan**
Created serviceUpdate.spec
Added updateServiceById and getServiceById in serviceHelper

### CODE REVIEW

**Review by RafailFerre — DISMISSED** [review]

**Review by ZdanovichOleg — DISMISSED** [review]
Well done!

**Comment by dashakhan** [comment]
@RafailFerre @ZdanovichOleg Hey guys, could you please take a look again, I had to resolve conflicts.Thanks!

**Review by ZdanovichOleg — DISMISSED** [review]

File: test/services/serviceUpdate.spec.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+describe('SERVICE UPDATE', () =>{
+  let service, vendorId, updateService, updatedData, getService
+  before(async() =>{
+    vendorId = (await vendorHelper.createVendor(vendorHelper.vendorData)).body.payload
+    updatedData = {...serviceHelper.serviceData(vendorId), name: 'Updated name', clientPrice: 0, vendorPrice: 0}
+    service = await serviceHelper.createService(serviceHelper.serviceData(vendorId))
+  })
+  after(async() =>{
+    await serviceHelper.deleteService(service)
```
Comment (dumpdusty) [inline test/services/serviceUpdate.spec.js]: why don't we delete created vendor as well?

**Review by dumpdusty — COMMENTED** [review]
please double check the comment

**Comment by dashakhan** [comment]
> please double check the comment

Thank you! Fixed

**Review by dumpdusty — DISMISSED** [review]
looks much better now!

**Review by RafailFerre — APPROVED** [review]

**Review by dumpdusty — APPROVED** [review]
Great job! There are a lot of tests!

**Comment by dashakhan** [comment]
> Great job! There are a lot of tests!

Ty! It was basically team work)

**Pull request merged**